resources/util.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+#
+# Define terminal colours
+#
+
+if [ -t 2 ]; then
```

Comment:
I had to look up what `[ -t 2]`. Could you please provide a comment explain why this is here? I reckon it's to define ANSI colours ONLY when `stderr` is outputting to the terminal (as opposed to redirected to a file).

Reply:
Sure :)